### ISSUE TITLE
check-inside-openshift fails with: psycopg2.errors.UndefinedTable: relation "copr_builds" does not exist

### ISSUE THREAD

**Issue by jpopelka** [packit-service/packit-service#750]
```
$ make check-inside-openshift PATH_TO_SECRETS=/home/jpopelka/git/packit-service/packit-service/secrets
...
self = <sqlalchemy.dialects.postgresql.psycopg2.PGDialect_psycopg2 object at 0x7fa88f432fa0>
cursor = <cursor object at 0x7fa88c3f25e0; closed: -1>
statement = 'DELETE FROM copr_builds', parameters = {}
context = <sqlalchemy.dialects.postgresql.psycopg2.PGExecutionContext_psycopg2 object at 0x7fa88bec95b0>
    def do_execute(self, cursor, statement, parameters, context=None):
>       cursor.execute(statement, parameters)
E       psycopg2.errors.UndefinedTable: relation "copr_builds" does not exist
E       LINE 1: DELETE FROM copr_builds
E                           ^
/usr/lib64/python3.8/site-packages/sqlalchemy/engine/default.py:593: UndefinedTable
The above exception was the direct cause of the following exception:
    @pytest.fixture()
    def clean_before_and_after():
>       clean_db()
tests_requre/conftest.py:161: 
```

http://pastebin.test.redhat.com/887158

**Comment by csomh** [comment]
Shouldn't this have been fixed by #748?

Sorry for the mess, I think my understanding of how JobTriggerModel works is very sloppy and gave the wrong advice for Anchit.

**Comment by jpopelka** [comment]
Yes, you're right. I must have forgotten to rebase. Works OK now. Closing.
